src/sage/schemes/elliptic_curves/hom_scalar.py

```diff
@@ -81,8 +81,8 @@
     sage: zero(E.random_point())
     (0 : 1 : 0)
 
-Due to a bug (:issue:`6413`), retrieving multiplication-by-`m` maps
-when `m` is divisible by the characteristic currently fails::
+    Retrieving multiplication-by-`m` maps when `m` is divisible by the
+    characteristic also works (since :issue:`37096`)::
 
```

Comment:
Unindent lines 84-85.

Reply:
Thank you. I should not resolve merge conflicts using the GitHub website, too easy to make silly mistakes.